server/middleware/measure.go

```diff
@@ -117,37 +117,51 @@ func measureStream() grpc.StreamServerInterceptor {
 }
 
 func (w *wrappedStream) RecvMsg(m interface{}) error {
+	err := w.ServerStream.RecvMsg(m)
+	if err != nil {
+		ulog.E(err)
```

Comment:
why not just return it from here?

Reply:
I had trouble with that, it breaks things. I wanted to do it like the SendMsg (it returns error there), but it even breaks the tests.